app/assets/javascripts/admin/addon/components/webhook-events.js

```diff
@@ -11,10 +11,14 @@ import I18n from "discourse-i18n";
 export default class WebhookEvents extends Component {
   @service messageBus;
   @service store;
+  @service dialog;
 
   @tracked pingEnabled = true;
   @tracked events = [];
   @tracked incomingEventIds = [];
+  @tracked failedEventCount = 0;
+  @tracked eventIds = [];
```

Comment:
It looks like neither `failedEventCount` nor `eventIds` need to be tracked here either. Both are re-calculated inside of functions, so you can use local variables (`const/let`) inside the functions. You only really need `@tracked` when the UI need to update state when the variable changes in certain cases.